src/infrastructure/delivery/provider/Mailchimp/Message.ts

```diff
@@ -0,0 +1,20 @@
+import {Message} from "../entities";
+import {MCMessage} from "./entities";
+
+export const mapMailchimpMessage = (message: Message): MCMessage => {
+    return {
+        html: message.message.html,
+        text: message.getPlainText(),
+        subject: message.message.subject,
+        from_email: message.message.from.address,
+        from_name: message.message.from.name,
+        to: message.message.to.map(to => {
+            return {
+                email: to.address,
+                name: to.name,
+                type: 'to'
+            }
+        })
+    }
+
+}
```

Comment:
Nits:

```suggestion
import {Message, MCMessage} from "./entities";

export const mapMailchimpMessage = (message: Message): MCMessage => ({
    html: message.message.html,
    text: message.getPlainText(),
    subject: message.message.subject,
    from_email: message.message.from.address,
    from_name: message.message.from.name,
    to: message.message.to.map(to => ({
          email: to.address,
          name: to.name,
          type: 'to'
   }))
})
```

Reply:
they are 2 different `entities` file. One for generic and one for provider (see the path).

Reply:
ah yes 👍🏼 sorry,(nit) but I think is nice to avoid the return instruction